nautobot/core/api/utils.py

```diff
@@ -337,9 +337,15 @@ def return_nested_serializer_data_based_on_depth(serializer, depth, obj, obj_rel
         obj_related_field_name: Object's field name that represents the related object.
     """
     if depth == 0:
-        result = obj_related_field.get_absolute_url(api=True)
+        url = obj_related_field.get_absolute_url(api=True)
         if serializer.context.get("request"):
-            result = serializer.context.get("request").build_absolute_uri(result)
+            url = serializer.context.get("request").build_absolute_uri(url)
+
+        result = {
+            "id": obj_related_field.pk,
+            "object_type": obj_related_field._meta.app_label + "." + obj_related_field._meta.model_name,
+            "url": url,
+        }
```

Comment:
Pretty sure this will necessitate some updates to the `docs/rest-api/overview.md`. :-)